Flank/src/main/java/com/walmart/otto/Flank.java

```diff
@@ -113,6 +112,24 @@ public static void main(String[] args) {
     }
   }
 
+  static OptionParser getOptionParser() {
+    return new OptionParser() {
```

Comment:
should we declare `p` as well?

Reply:
oh, great catch. I will fix asap.
I don't use -p so when I run this version with my env I didn't find it. My bad.

Reply:
I just fixed it. Thanks again.

Reply:
Cool, let's wait for @renas then :)